great_expectations/profile/user_configurable_profiler.py

```diff
@@ -911,16 +911,7 @@ def _build_expectations_primary_or_compound_key(
             len(column_list) > 1
             and "expect_compound_columns_to_be_unique" not in self.excluded_expectations
         ):
-            if isinstance(profile_dataset, Validator) and not hasattr(
-                profile_dataset, "expect_compound_columns_to_be_unique"
-            ):
-                # TODO: Remove this upon implementation of this expectation for V3
-                logger.warning(
-                    "expect_compound_columns_to_be_unique is not currently available in the V3 (Batch Request) API. Specifying a compound key will not add any expectations. This will be updated when that expectation becomes available."
-                )
-                return profile_dataset
-            else:
-                profile_dataset.expect_compound_columns_to_be_unique(column_list)
+            profile_dataset.expect_compound_columns_to_be_unique(column_list)
```

Comment:
Is this why you changed the default value of catch_exceptions? If so, just add catch_exceptions explicitly here.

Reply:
No, that was a typo.  This code was changed because the expectation is now implemented (as per the TODO comments).

Reply:
Yay!